### PR TITLE
fix: bullet points for install page of monitoring/dynatrace corrected

### PR DESCRIPTION
**Pull request by j-poecher**
This PR fixes the bullet points of the **Notes** section of page `monitoring/dynatrace/install` for versions `0.11.x`, `0.12.x` and `0.13.x`

### CODE REVIEW

File: content/docs/0.11.x/monitoring/dynatrace/install/index.md [hunk 1]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
-- To ensure that existing workloads are being monitored, please restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
-  ```console
-  kubectl delete pods --all --namespace=sockshop-dev
-  ```
+
+* By default, most Kubernetes clusters will only offer a self-signed certificate. In such cases, please select *Skip SSL Security Check* when deploying the Dynatrace Operator.
+
+* When deploying the Dynatrace Operator to a cluster running on a *Container-Optimized OS (COS)*, which includes GKE, Anthos, CaaS and PKS environments, please select the *Enable volume storage* option.
+
+* To verify that the Dynatrace Operator is working properly, check the status of the pods, e.g. using kubectl:
```
Comment (StackScribe) [inline content/docs/0.11.x/monitoring/dynatrace/install/index.md:20]:
```suggestion
* To verify that the Dynatrace Operator is working properly, check the status of the pods with the kubectl command:
```

File: content/docs/0.11.x/monitoring/dynatrace/install/index.md [hunk 2]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
-  ```console
-  kubectl get pods -n dynatrace
-  ```
-  A status of `Running` indicates that the Dynatrace Operator is functioning normally, while `Error` or `CrashLoopBackOff` are signs of a problem.
-- To ensure that existing workloads are being monitored, please restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
-  ```console
-  kubectl delete pods --all --namespace=sockshop-dev
-  ```
+
+* By default, most Kubernetes clusters will only offer a self-signed certificate. In such cases, please select *Skip SSL Security Check* when deploying the Dynatrace Operator.
```
Comment (StackScribe) [inline content/docs/0.11.x/monitoring/dynatrace/install/index.md:16]:
```suggestion
* By default, most Kubernetes clusters only offer a self-signed certificate. In such cases, please select *Skip SSL Security Check* when deploying the Dynatrace Operator.
```

File: content/docs/0.11.x/monitoring/dynatrace/install/index.md [hunk 3]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
+
+* To verify that the Dynatrace Operator is working properly, check the status of the pods, e.g. using kubectl:
+
+    A status of `Running` indicates that the Dynatrace Operator is functioning normally, while `Error` or `CrashLoopBackOff` are signs of a problem.
+
+    ```console
+    kubectl get pods -n dynatrace
+    ```
+
+* To ensure that existing workloads are being monitored, please restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
```
Comment (StackScribe) [inline content/docs/0.11.x/monitoring/dynatrace/install/index.md:28]:
```suggestion
* To ensure that existing workloads are being monitored, restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
```

File: content/docs/0.12.x/monitoring/dynatrace/install/index.md [hunk 4]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
 To setup Dynatrace monitoring on your Kubernetes cluster, please follow the official Dynatrace documentation on [Deploy Dynatrace Operator](https://www.dynatrace.com/support/help/technology-support/container-platforms/kubernetes/monitor-kubernetes-environments/).
```
Comment (StackScribe) [inline content/docs/0.12.x/monitoring/dynatrace/install/index.md:12]:
```suggestion
To set up Dynatrace monitoring on your Kubernetes cluster, please follow the official Dynatrace documentation on [Deploy Dynatrace Operator](https://www.dynatrace.com/support/help/technology-support/container-platforms/kubernetes/monitor-kubernetes-environments/). 
```
"setup" is a noun or adjective; the verb is "set up"

File: content/docs/0.12.x/monitoring/dynatrace/install/index.md [hunk 5]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
-  ```console
-  kubectl get pods -n dynatrace
-  ```
-  A status of `Running` indicates that the Dynatrace Operator is functioning normally, while `Error` or `CrashLoopBackOff` are signs of a problem.
-- To ensure that existing workloads are being monitored, please restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
-  ```console
-  kubectl delete pods --all --namespace=sockshop-dev
-  ```
+
+* By default, most Kubernetes clusters will only offer a self-signed certificate. In such cases, please select *Skip SSL Security Check* when deploying the Dynatrace Operator.
```
Comment (StackScribe) [inline content/docs/0.12.x/monitoring/dynatrace/install/index.md:16]:
```suggestion
* By default, most Kubernetes clusters only offer a self-signed certificate. In such cases, please select *Skip SSL Security Check* when deploying the Dynatrace Operator.
```

File: content/docs/0.12.x/monitoring/dynatrace/install/index.md [hunk 6]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
+
+* When deploying the Dynatrace Operator to a cluster running on a *Container-Optimized OS (COS)*, which includes GKE, Anthos, CaaS and PKS environments, please select the *Enable volume storage* option.
+
+* To verify that the Dynatrace Operator is working properly, check the status of the pods, e.g. using kubectl:
```
Comment (StackScribe) [inline content/docs/0.12.x/monitoring/dynatrace/install/index.md:20]:
```suggestion
* To verify that the Dynatrace Operator is working properly, check the status of the pods by using the kubectl command:
```

File: content/docs/0.12.x/monitoring/dynatrace/install/index.md [hunk 7]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
+
+    A status of `Running` indicates that the Dynatrace Operator is functioning normally, while `Error` or `CrashLoopBackOff` are signs of a problem.
+
+    ```console
+    kubectl get pods -n dynatrace
+    ```
+
+* To ensure that existing workloads are being monitored, please restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
```
Comment (StackScribe) [inline content/docs/0.12.x/monitoring/dynatrace/install/index.md:28]:
```suggestion
* To ensure that existing workloads are being monitored, restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
```

File: content/docs/0.13.x/monitoring/dynatrace/install/index.md [hunk 8]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
-  ```console
-  kubectl get pods -n dynatrace
-  ```
-  A status of `Running` indicates that the Dynatrace Operator is functioning normally, while `Error` or `CrashLoopBackOff` are signs of a problem.
-- To ensure that existing workloads are being monitored, please restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
-  ```console
-  kubectl delete pods --all --namespace=sockshop-dev
-  ```
+
+* By default, most Kubernetes clusters will only offer a self-signed certificate. In such cases, please select *Skip SSL Security Check* when deploying the Dynatrace Operator.
```
Comment (StackScribe) [inline content/docs/0.13.x/monitoring/dynatrace/install/index.md:16]:
```suggestion
* By default, most Kubernetes clusters only offer a self-signed certificate. In such cases, please select *Skip SSL Security Check* when deploying the Dynatrace Operator.
```

File: content/docs/0.13.x/monitoring/dynatrace/install/index.md [hunk 9]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
+
+* When deploying the Dynatrace Operator to a cluster running on a *Container-Optimized OS (COS)*, which includes GKE, Anthos, CaaS and PKS environments, please select the *Enable volume storage* option.
+
+* To verify that the Dynatrace Operator is working properly, check the status of the pods, e.g. using kubectl:
```
Comment (StackScribe) [inline content/docs/0.13.x/monitoring/dynatrace/install/index.md:20]:
```suggestion
* To verify that the Dynatrace Operator is working properly, check the status of the pods by using the kubectl command:
```

File: content/docs/0.13.x/monitoring/dynatrace/install/index.md [hunk 10]
```diff
@@ -12,18 +12,24 @@ Bring your Dynatrace SaaS or Dynatrace-managed tenant. If you do not have a Dyna
...
+
+    A status of `Running` indicates that the Dynatrace Operator is functioning normally, while `Error` or `CrashLoopBackOff` are signs of a problem.
+
+    ```console
+    kubectl get pods -n dynatrace
+    ```
+
+* To ensure that existing workloads are being monitored, please restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
```
Comment (StackScribe) [inline content/docs/0.13.x/monitoring/dynatrace/install/index.md:28]:
```suggestion
* To ensure that existing workloads are being monitored, restart the pods. For example, to restart the pods in the `sockshop-dev` namespace:
```

**Review by StackScribe — APPROVED** [review]
A few editorial suggestions, none of which are show-stoppers

**Review by oleg-nenashev — APPROVED** [review]
Thanks!

**Pull request merged**